README.md

```diff
@@ -0,0 +1,53 @@
+# StickyHeadroom
+[![npm badge](https://img.shields.io/npm/v/@integreat-app/sticky-headroom.svg)](https://www.npmjs.com/package/@integreat-app/sticky-headroom)
+
+StickyHeadroom is a React component, that hides your header when you scroll down and shows it
+once you're scrolling up again.
+It's designed for best performance and can only be used, when you know the height of your header
+component (or more precisely the amount of pixels you want StickyHeadroom to hide).
+This helps us avoid calculating the height ourselves and therefore browsers don't need to perform
+heavy Recalculate-Style-Phases.
+For more information read [here](https://developers.google.com/web/fundamentals/performance/rendering/).
+
+Since it's using [styled-components](https://www.styled-components.com/) internally, it's best to 
+use it in apps where you already have styled-components in place.
+
+The component is inspired by [react-headroom](https://kyleamathews.github.io/react-headroom/).
+
+## Usage
+A basic usage example:
+```jsx
+render () {
+  return <StickyHeadroom scrollHeight={100} />
+    <div style={{height: '100px', backgroundColor: 'red'}}>MyHeader</div>
+  </StickyHeadroom>
+}
+```
+
+## API
+You can pass the following props to StickyHeadroom:
+* `children: React.Node` The header component, that should be hidden and revealed
+* `scrollHeight: number` The maximum amount of px the header should move up when scrolling
+* `pinStart: number` The minimum scrollTop position where the transform should start
+* `stickyAncestor?: Element<*>` (Optional, Deprecated) Is rendered with a corresponding stickyTop prop as an ancestor node. Requires `height` prop to be specified.
```

Comment:
Is there a reason why something that's already deprecated should be included in the first version of the package?

Reply:
Removed now